### DISCUSSION TITLE
JWK Endpoint not validated by web3auth when creating a custom auth

### DISCUSSION THREAD

**Discussion by Myrindd** [Web3Auth/Web3Auth#1127] [Help]
Hi,
I am trying to create a custom auth for web3auth; but the dashboard refuse to validate the following link:
[enpoint link](https://alpha.kipengine.dev/.well-known/jwks.json)

I tried testing the endpoint on jwt.davetonge.co.uk using a JWT, and it seemed fine.
Can you tell me what is wrong?
Thank you!

**Comment by shahbaz17** [comment]
Hey @Myrindd 

Could you share more details here?

Is this by team KIP?

**Comment by Myrindd** [comment]
![image](https://user-images.githubusercontent.com/67431109/209697709-2bb01c2d-bf9c-4c2b-8d71-5bcbd2019e8e.png)

**Reply by shahbaz17** [reply]
Please update to the latest packages.

Fix has been released for this.
